### PR TITLE
Configuration parser

### PR DESCRIPTION
**Pull request by saleph**
Working skeleton of the parser. Still needs more wrapping (more advanced classes holding triggers and logs types and configuration parser) but the main functionality is done.

In relation to: #5 #6

### CODE REVIEW

File: flex/configparser.l [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+\n+					/* consume whitespaces at the beggining */
+{log}{ws}+			yy_push_state(logEnv);
+{trigger}{ws}+		yy_push_state(triggerEnv);
+<<EOF>>				yyterminate();
+					
+
+%%
+
+
+//int main( int /* argc */, char** /* argv */ )
```
Comment (poskart) [inline flex/configparser.l]: Should be deleted if not used

File: include/configrawdata.hpp [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#ifndef INCLUDE_CONFIGRAWDATA_HPP_
+#define INCLUDE_CONFIGRAWDATA_HPP_
+
+/// Structure describing resource value of particular trigger.
```
Comment (poskart) [inline include/configrawdata.hpp:4]: What about this triple '///'. Maybe:
/*
 *
*/
is better?

Comment (saleph) [reply include/configrawdata.hpp:4]: Tripple /// is used for doxygen's auto-generated docs.
In C++ is suggested to use // coments (no problem with accidental comment spread).

File: flex/configparser.l (hunk 1 above)
Comment (saleph) [reply flex/configparser.l]: Done

File: include/configrawdata.hpp (hunk 2 above)
Comment (poskart) [reply include/configrawdata.hpp:4]: Oks

**Pull request merged**